src/lib/openapi/spec/search-events-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const searchEventsSchema = {
+    $id: '#/components/schemas/searchEventsSchema',
+    type: 'object',
+    properties: {
+        type: {
+            type: 'string',
+        },
+        project: {
+            type: 'string',
+        },
+        feature: {
```

Comment:
should add a description that this is the name of the feature toggle, and that we assume the correct CasingOfToggle.